### PR TITLE
TDP-4435 Add relative url notes everywhere they're relevant in docs

### PR DESCRIPTION
**Pull request by CatexB**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4435.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/verbs/playAudio.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ header is set on the response, media will not be cached.
 ### Text Content
 | Name     | Description                                                                                                                                  |
 |:---------|:---------------------------------------------------------------------------------------------------------------------------------------------|
-| audioUri | The URL of the audio file realative to the endpoint that returned the BXML <br> ⚠️ **ONLY** .wav files encoded as PCM or G711 are supported. |
+| audioUri | The URL of the audio file relative to the endpoint that returned the BXML <br> ⚠️ **ONLY** .wav files encoded as PCM or G711 are supported. |
```
Comment (n-miles) [inline voice/bxml/verbs/playAudio.md]: This is confusing. Can we make the wording more similar to the wording for callbacks? Like "The URL of the audio file. May be a relative URL." or similar?

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4435.s3-website-us-east-1.amazonaws.com/

**Review by n-miles — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-4435.s3-website-us-east-1.amazonaws.com/

**Review by marcelohossomi — APPROVED** [review]

**Pull request merged**